src/Spinner/Spinner.tsx

```diff
@@ -1,41 +1,50 @@
 import React from 'react'
 import styled from 'styled-components'
 import sx, {SxProp} from '../sx'
-import {ComponentProps} from '../utils/types'
+import {Status} from '../internal/components/LiveRegion'
 
 const sizeMap = {
   small: '16px',
   medium: '32px',
   large: '64px',
 }
 
-export interface SpinnerInternalProps {
+export type SpinnerProps = React.PropsWithChildren<{
+  /**
+   * Provide a message to be announced to screen readers while the current
+   * context is loading
+   */
+  loadingMessage?: string
+
   /** Sets the width and height of the spinner. */
   size?: keyof typeof sizeMap
-}
+}>
 
-function Spinner({size: sizeKey = 'medium', ...props}: SpinnerInternalProps) {
+function Spinner({loadingMessage, size: sizeKey = 'medium', ...props}: SpinnerProps) {
   const size = sizeMap[sizeKey]
 
   return (
-    <svg height={size} width={size} viewBox="0 0 16 16" fill="none" {...props}>
-      <circle
-        cx="8"
-        cy="8"
-        r="7"
-        stroke="currentColor"
-        strokeOpacity="0.25"
-        strokeWidth="2"
-        vectorEffect="non-scaling-stroke"
-      />
-      <path
-        d="M15 8a7.002 7.002 0 00-7-7"
-        stroke="currentColor"
-        strokeWidth="2"
-        strokeLinecap="round"
-        vectorEffect="non-scaling-stroke"
-      />
-    </svg>
+    <>
+      {loadingMessage !== undefined ? <Status message={loadingMessage} /> : null}
```

Comment:
> So Status will buffer the announcement so that it doesn't get missed

Exactly 💯 when we run into a case where a live region is not on the page yet, we have make a `live-region` custom element that buffers announcements until it is in the DOM.

Thankfully this should help to address issues with both creating the live region and having a message at the same time by separating them 🤞